content/about/myths.md

```diff
@@ -21,6 +21,36 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
 
 But even ignoring the high performance, specialist end, XML still holds its own. Consider [these findings](https://www.balisage.net/Proceedings/vol10/html/Lee01/BalisageVol10-Lee01.html), for example.
 
+## XMPP is not secure.
+
+The hypothesis:
+> XMPP is not secure.
+
+The fact:
+> XMPP offers state-of-the-art end-to-end encryption, and a relatively high-degree of pseudonymity
+
+"Secure" is not an absolute. In order to decide whether XMPP is secure, you need to decide what you want to be secure against. No system that relies on servers is secure against for example a rogue administrator (end-to-end encryption can protect your message contents even against a rogue admin, but only if you verify keys appropriately). Instead, our specifications consistently remind developers of the risks of various choices. Plenty of XMPP extensions will certainly increase the risk for both for the users and for the administrator - including allowing registration and password changing via the protocol.
+
+Different deployment patterns radically alter the security, too. XMPP is designed by its nature to be highly flexible. This is why it is used by everything from computer games to military and nation states, and from telephone switchboards to the video game handhelds. Security that suits the military will rarely suit the average internet user, and vice-versa - XMPP strives to handle all of these, and it's certainly possible to configure a server to provide much worse security - or much better. That said, basic security is both encouraged and provided for; and this is most effective when using a server you trust. One way of trusting your server administrator is, of course, to be your own server administrator. This may sound obvious, but XMPP has from its outset ensured that this is always an option; even on low-budget.
```

Comment:
> from computer games to military and nation states

Sounds like you want to describe the scale here... so I'd rather write "from gamers to militaries and nation states".

> and from telephone switchboards to the video game handhelds

You can drop `the`.